presto-orc/src/test/java/io/prestosql/orc/OrcTester.java

```diff
@@ -358,11 +358,11 @@ private void testStructRoundTrip(Type type, List<?> values)
             Type readType = rowType(type, type, type, type, type, type);
             Type writeType = rowType(type, type, type);
 
-            List writeValues = values.stream()
+            List<?> writeValues = values.stream()
                     .map(OrcTester::toHiveStruct)
                     .collect(toList());
 
-            List readValues = values.stream()
+            List<?> readValues = values.stream()
```

Comment:
You may want to fix usage of rawtypes in ParquetTester as well

Reply:
I added another commit for `ParquetTester`